go.sum

```diff
@@ -253,8 +253,8 @@ github.com/nsf/jsondiff v0.0.0-20200515183724-f29ed568f4ce h1:RPclfga2SEJmgMmz2k
 github.com/nsf/jsondiff v0.0.0-20200515183724-f29ed568f4ce/go.mod h1:uFMI8w+ref4v2r9jz+c9i1IfIttS/OkmLfrk1jne5hs=
 github.com/oklog/run v1.0.0 h1:Ru7dDtJNOyC66gQ5dQmaCa0qIsAUFY3sFpK1Xk8igrw=
 github.com/oklog/run v1.0.0/go.mod h1:dlhp/R75TPv97u0XWUtDeV/lRKWPKSdTuV0TZvrmrQA=
-github.com/packethost/packngo v0.14.1 h1:hufjiwnWrxuI1iolRt/ZQw8E8YQeTDGIjB8vjELYGlY=
-github.com/packethost/packngo v0.14.1/go.mod h1:YrtUNN9IRjjqN6zK+cy2IYoi3EjHfoWTWxJkI1I1Vk0=
+github.com/packethost/packngo v0.17.0 h1:fGPlj9NDt6ejOrAMfUgx955oCaR1QBKA9pec14m0L38=
+github.com/packethost/packngo v0.17.0/go.mod h1:YrtUNN9IRjjqN6zK+cy2IYoi3EjHfoWTWxJkI1I1Vk0=
```

Comment:
This can be picked up in a rebase (we also vendor/, which the rebase will pick up)

Reply:
To confirm: You want me to remove the dependency changes for now?

Reply:
`main` already has the dependency change. Just rebase and you will get it for free via #151.

The other dependency changes (the vendoring of new sdk customize .go files) seems important to keep in this PR.